uncertainty_mapping/README.md

```diff
@@ -0,0 +1,22 @@
+# Uncertainty mapping visualisation
+
+This component processes input mcap (ROSbags) and produces a map.html file.
+Opening the map.html file in a browser shows the AV GPS trace and draws ellipses
+representing the covariance (uncertainty) of the AV location.
+
+## Usage
+Run script to build docker image and access interactive container:
+
+```
+./dev.sh -p /path/to/mcap
```

Comment:
The `-p` flag expects a directory rather than a `mcap` bag. I'd change it to

``` bash
./dev.sh -p /path/to/your/rosbags
```

Just to be super clear :)

Reply:
I see your Readme fixes and I raise you some script fixes :dollar: